TreeTracker/UI/Views/Home/HomeViewController.swift

```diff
@@ -81,9 +81,10 @@ class HomeViewController: UIViewController, AlertPresenting {
         }
     }
 
-    override func viewDidLoad() {
-        super.viewDidLoad()
+    override func viewWillAppear(_ animated: Bool) {
```

Comment:
Looking at this I have named the `fetchTrees()` badly as it doesn't reflect what it does accurately enough, rather than performing a single fetch it starts a service that uses an `NSFetchedResultsController` which continuously monitors core data and updates the delegate with changes. 

Because of this we would only want to kick off `fetchTrees()` once, so I would be inclined to leave it in `viewDidLoad` as `viewWillAppear` will get called multiple times.

It makes sense for your `fetchProfileData()` call to be called here though 👍